src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -417,7 +416,6 @@ internal static int GetWarningLevel(ErrorCode code)
                 case ErrorCode.WRN_NullReferenceArgument:
                 case ErrorCode.WRN_NullabilityMismatchInTypeOnOverride:
                 case ErrorCode.WRN_NullabilityMismatchInReturnTypeOnOverride:
-                case ErrorCode.WRN_NullabilityMismatchInReturnTypeOnPartial:
                 case ErrorCode.WRN_NullabilityMismatchInParameterTypeOnOverride:
                 case ErrorCode.WRN_NullabilityMismatchInParameterTypeOnPartial:
```

Comment:
Should we also delete the `ErrorCode.WRN_NullabilityMismatchInParameterTypeOnPartial`?

Reply:
I think I get it now 😉 we are introducing different rules for extended versus regular partial methods, and thus we will never encounter acceptable nullability differences in partial method returns, but we may encounter acceptable nullability differences in partial method parameters.